optimum/exporters/onnx/model_configs.py

```diff
@@ -507,6 +507,14 @@ class GemmaOnnxConfig(LlamaOnnxConfig):
     MIN_TRANSFORMERS_VERSION = version.parse("4.38.0")
 
 
+@register_tasks_manager_onnx("gemma2", *[*COMMON_TEXT_GENERATION_TASKS, "text-classification"])
+class Gemma2OnnxConfig(TextDecoderOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedTextConfig
+    DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, GemmaDummyPastKeyValuesGenerator)
+    DUMMY_PKV_GENERATOR_CLASS = GemmaDummyPastKeyValuesGenerator
+    MIN_TRANSFORMERS_VERSION = version.parse("4.51.0")
```

Comment:
I think it should be lower

Reply:
```suggestion
    MIN_TRANSFORMERS_VERSION = version.parse("4.45.0")
```

Reply:
gemma2 was added in v4.42 but never supported tuple for pkv, `DynamicCache` officially supported since 4.53 so setting it to the min transformers version in https://github.com/huggingface/optimum-onnx/pull/32/commits/6c7235043aceb648154832c53d89bd6c1a034dac. We could technically set the min transformers version to a lower version  but would need some additional tests to make sure the model is correctly exported (modeling initially using `HybridCache` which is now deprecated), let me know if that works for you @IlyasMoutawwakil !